contracts/PoolManager.sol

```diff
@@ -108,6 +108,25 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC1155, IERC1155Rec
         return pools[id].positions.get(owner, tickLower, tickUpper);
     }
 
+    function _accountNoOp(PoolKey memory key, BalanceDelta hookDelta) internal {
```

Comment:
nit: this function would be more well organized below `_accountPoolBalanceDelta` or at least in that general vicinity